packages/transformers/js/core/src/hoist.rs

```diff
@@ -642,6 +642,9 @@ impl<'a> Fold for Hoist<'a> {
         // require('foo') -> $id$import$foo
         if let Some(source) = match_require(&node, &self.collect.decls, self.collect.ignore_mark) {
           self.add_require(&source);
+
+          // !!
+
```

Comment:
What we need here is the logic from the old scope hoisting implementation which checked `isUnused()`, so if the return value is unused (`require("x");` is a statement or as the non-last item in a sequence expression)

Reply:
Why do we need to know if the return value is unused? Don't we still need to include the code for side effects?

Reply:
Yes, but the `'*'` symbol shouldn't be added.

Reply:
Oh I see. Well it probably shouldn't crash even if you did something like `var x = require('./x.css')`... Don't we do something in the non scope hoisting packager to handle adding an empty JS module when you imported a non-JS asset?